posts/share-cpp/index.qmd

```diff
@@ -184,7 +184,7 @@ void print_hello_world() {
 
 ::: {.callout-tip}
 Remember to add `PKG_CPPFLAGS += -I../inst/include/` to both `Makevars` and `Makevars.win` under `src/`.
-Furthermore, as noted, the package will not automatically cause a rebuild when headers are modified --- this needs to be done manually.
+Furthermore, as noted in the [Rcpp attributes documentation](https://dirk.eddelbuettel.com/code/rcpp/Rcpp-attributes.pdf), the package will not automatically cause a rebuild when headers are modified --- this needs to be done manually.
 :::
 
 ## Linking header code between pacakges
```

Comment:
Typo in line 197 (specifcy)?

Reply:
Fixed in https://github.com/epiverse-trace/epiverse-trace.github.io/pull/34/commits/19f9fea49dfa48aa52d007cde29a79615be7d630